Add config option to change the visibility of the switcher (#70)

funnyboy-roks opened the pull request:
This addresses #70.

Adds a config option, `switcher_visibility`, which allows the user to switch between `"visible"`, `"hidden"`, `"F1"`, ..., `"F12"`, for always visible, always hidden, and a key to toggle the visibility.  

When `switcher_visibility` is set to a key, a hint appears in the top-left corner, under the shutdown/reboot hints, by default, the format for the hint is `"Switcher %key%"`.

When `switcher_visibiltiy` is set to `"hidden"`, cycling through input fields skips over the switcher, to prevent it from being selected/modified.

coastalwhite commented:
I really, really like the idea of this, but I am not 100% sure of the positioning in the UI. Is there any chance the UI can be moved to the right top corner in the same chunk as the power_menu?

Again, thank you very much. Would love to merge this.

funnyboy-roks commented:
Good idea.  I've not used `tui` crate before, but I'd be very surprised if that's not possible. I'll take a look at moving it to the right after this coming weekend.

funnyboy-roks commented:
It turns out that I did have time to do it today.  It's really quite a simple thing of adding `.align(Alignment::Right)` to the paragraph.

In order to keep them o the same line, I changed the name of `PowerMenuWidget` to `KeyMenuWidget` and removed the `SwitcherToggleMenuWidget`, merging it into the `KeyMenuWidget`

coastalwhite commented:
Sorry, this got lost a bit in my GitHub messages. I resolved some small issues and am happy to merge this now. Thank you very much for the PR :smile:

funnyboy-roks commented:
Wonderful, thank you!
